### PR TITLE
Features branch

### PR DESCRIPTION
**Pull request by nicupop729**
## For this app, project requirements were followed and the next features were applied:

When the page loads, the web app shows the list of continents.
After the user chooses a continent, all the countries from that continent are displayed.
The user now can choose a country to receive information about it.
Anywhere in the app, the user can use the search bar, which will filter from 195 countries to display the country he searches for.

Technical set up:

Set up the repository on GitHub and use Gitflow;
Set up React and Redux;
Set up testing libraries;
We do not preserve user data;
The project is a single page application (SPA) built with React and Redux.
The data retrieved from the API is stored in the Redux store.
Every page (the main page and the pages for each item) have a unique route within the SPA.

[link to presentations video](https://www.loom.com/share/03a1fe8bbcb347efa46a8a77d4b30cd1)

### CODE REVIEW

**Comment by nicupop729** [comment]
https://world-countries-react-redux.netlify.app/   -  deploy link on netlify

File: src/components/countries/Countries.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+import asiaMap from '../../assets/map_Asia.svg.png';
+import africaMap from '../../assets/map-Africa.svg.png';
+import oceaniaMap from '../../assets/map_Oceania.svg.png';
+import northAmericaMap from '../../assets/map-North_America-Subdivisions.svg.png';
+import centralAmericaMap from '../../assets/map_America_Central-color.svg.png';
+import southAmericaMap from '../../assets/map_of_South_America.svg.png';
+import './countries.css';
+import DisplayCountries from './DisplayCountries';
+
+const Countries = ({
```
Comment (TedLivist) [inline src/components/countries/Countries.js:13]: - [ ] As stated in the project requirements, the **`Details`** (here Countries) page should have a 'Back' button (<) which navigates the user to the **`Homepage`**. Kindly add this button as contained in the design template.

File: src/components/country/Country.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
+import PropTypes from 'prop-types';
+import { v4 as uuidv4 } from 'uuid';
+import DisplayCountries from '../countries/DisplayCountries';
+import './country.css';
+
+const Country = ({
```
Comment (TedLivist) [inline src/components/country/Country.js:8]: - [ ] **[OPTIONAL]** It would be nice to add a 'Back' button to the **`Country`** page as well.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @nicupop729,

Good job so far!
✔️ Good video presentation
✔️ Nice UI design
✔️ Clean code
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] While you added unit tests covering your components you are still missing integration tests using the **React Testing Library** as required in the project guidelines. Kindly add these. You can refer to this [documentation](https://testing-library.com/docs/react-testing-library/intro/) and the recommended [tutorial](https://www.robinwieruch.de/react-testing-library/) at the bottom of the article to get started. 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Nicu, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏
## Please go ahead and merge the branch!
## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**